mgmtsystem/google.py

```diff
@@ -537,8 +537,14 @@ def all_vms(self):
                     ))
         return result
 
-    def vm_creation_time(self, istance_name):
-        instance = self._find_instance_by_name(istance_name)
+    def vm_creation_time(self, instance_name):
+        instance = self._find_instance_by_name(instance_name)
         vm_time_stamp = instance['creationTimestamp']
         creation_time = (iso8601.parse_date(vm_time_stamp)).replace(tzinfo=None)
         return creation_time
+
+    def vm_type(self, instance_name):
+        instance = self._find_instance_by_name(instance_name)
+        machine_type = instance['machineType'].split('/')
```

Comment:
If it's not there, what will we get, just thinking that the split may fail on None?